README.md

```diff
@@ -27,6 +29,15 @@ The porting guide is available on [this page](http://www.FreeRTOS.org/FreeRTOS-P
 
 ## Repository structure
 This repository contains the FreeRTOS-Plus-TCP repository and a number of supplementary libraries for testing/PR Checks.
+Below is the breakdown of what each directory contains:
+- tools
+    - This directory contains the tools and related files (CMock/uncrustify) required for tests/checks for the TCP source code.
```

Comment:
Minor correction in grammar "This directory contains the tools and related files (CMock/uncrustify) required to run  tests/checks on the TCP source code"

Reply:
Thanks @gitwallit, I have updated the readme file.
Can you please re-review it?